tools/atip/atip/web/web.py

```diff
@@ -209,6 +209,23 @@ def __check_normal_text(self, text, display=True):
             print "Failed to get element: %s" % e
         return None
 
```

Comment:
web.py is a public library for all BDD tests, every change may affect other tests, so we should be very careful on web.py update. 
Please put the new add/update function into $suite/../steps.py or you can create a new web.py under the test suite.
And finally merge them after a comprehensive review and improvement.